core/buidler.config.js

```diff
@@ -0,0 +1,26 @@
+usePlugin("@nomiclabs/buidler-truffle5");
+usePlugin("solidity-coverage");
+
+module.exports = {
+  solc: {
+    version: "0.6.6",
+    optimizer: {
+      enabled: false,
```

Comment:
Shouldn't this be true? Otherwise, we're not testing against the optimized code we're actually running on mainnet, right? We can always change this when testing locally if we want stack traces. We could also make it depend on a CLI arg (like `--debug`). Thoughts?

Reply:
The debug flag sounds good! We'll optimize it by default unless a `--debug` flag is provided

Reply:
SGTM!